planner/core/stats.go

```diff
@@ -363,6 +363,15 @@ func (ds *DataSource) isInIndexMergeHints(name string) bool {
 	return false
 }
 
+// If AccessConds is empty or tableFilter is not empty, we ignore the access path.
+// Now these conditions are too strict.
+// For example, a sql `select * from t where a > 1 or (b < 2 and c > 3)` and table `t` with indexes
+// on a and b separately. we can generate a `IndexMergePath` with table filter `a > 1 or (b < 2 and c > 3)`.
+// TODO: solve the above case
+func (ds *DataSource) ConvertToIndexMergePath(path *util.AccessPath) bool {
```

Comment:
This function and its name are pretty confusing, why do we need this?

Reply:
OK.Because there are two places (line 390 and  line 416) that need to add

Reply:
Do you have any suggestions about the format?

Reply:
I think just adding this comment in one of the check places is fine, we don't need this empty function actually.

Reply:
@eurekaka empty function removed now.